logspout/start.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+
+tag_csv=logspout
+if [[ "${audius_loggly_tags}" ]]; then
+   tag_csv=${tag_csv},${audius_loggly_tags}
+fi
+
+hostname=$(echo ${audius_discprov_url} | sed -e 's/[^/]*\/\/\([^@]*@\)\?\([^:/]*\).*/\2/')
```

Comment:
content node won't have audius_discprov_url, there might be a content node equivalent

Reply:
Good catch. I didn't see anything implemented from the content node rsyslog implementation, but will look into that for this PR.

Reply:
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/config.js#L388